build/xunit.runner.visualstudio.desktop.props

```diff
@@ -11,6 +11,13 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <Visible>False</Visible>
     </Content>
+    <Content Include="$(MSBuildThisFileDirectory)..\_common\xunit.runner.reporters.desktop.dll"
+             Condition=" '$(TargetFrameworkVersion)' != 'v2.0' and '$(TargetFrameworkVersion)' != 'v3.0' and '$(TargetFrameworkVersion)' != 'v3.5' " 
```

Comment:
I actually attempted to research this....and in tens of pages of results, there's not a single mention of v3.5.1 as a TFV:

https://github.com/search?p=18&q=%3CTargetFrameworkVersion%3Ev3.5%3C%2FTargetFrameworkVersion%3E&type=Code&utf8=%E2%9C%93

I think we're safe :)